install_bootstrap

```diff
@@ -275,6 +276,10 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
       fc_lambda_domain='fc_lambda_domain'
       ask_question "What is the domain of the bagit-firecloud-lambda?" "$FC_LAMBDA_DOMAIN" "FireCloud Bagit Lambda Domain" $fc_lambda_domain
 
+      azul_s3_bucket='azul_s3_bucket'
+      ask_question "What is the name of the AWS S3 bucket where BDBag for Export-to-FireCloud are stored?" "$AZUL_S3_BUCKET" "Azul S3 Bucket" $azul_s3_bucket
```

Comment:
I change it, don't mention FireCloud, but added that the lifecycle is 1 d. It's okay to not mention FireCloud, but we don't want to give the user the impression that this is a permanent storage place for the bag. What do you think?

Reply:
I moved the Jira story. It's now in https://ucsc-cgl.atlassian.net/browse/DEP-24. I will mention it when I merge it.

Reply:
1. I wouldn't mention that the lifecycle is one day here. We may end up experimenting with this, e.g., a security consultant could say it should only be 30 minutes. It will evolve independently from this script, so I would only speak of it in general terms. Since it's an install script, you probably don't want to get into too much detail; perhaps you could add more information to the README. In fact, you probably should, now that I think about it, since you updated the README to discuss other settings that you are prompted for.
2. You should refer to the GitHub issue in your commit message. Your GitHub issue is #49 , and you can refer to it like that, `#49`.

Reply:
*response to 1*
I'm not sure we can set the lifecycle to less than 1 d? See https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-lifecycle.html. I suppose we can trigger a cron job if this becomes critical.

*response to 2*
I'll do it when I merge it. Or do you mean in every commit?

Reply:
1. That was just an example; maybe we should set the lifecycle to 1 week, or 3 days. My point is that the we control the lifecycle separately, via AWS console, and the value may change, so I don't think the install script should tie itself to a particular time that might change. In the README maybe you want to say it's currently 1 day, but I wouldn't have that level of detail for something that could change in the installer.
2. Yes, on the final commit when you squash and merge is what I meant. I was just pointing out that in GitHub you typically reference the GitHub issue, not the JIRA issue, and showing the markdown syntax to reference a GitHub issue.

Reply:
1. Okay, I took it out the time. In the end it's up to the user. And you bring up the right point: the user could be expected to know of the lifecycle issue as it is described in the README.
2. Got it. Will do.